src/app/services/agile.services.ts

```diff
@@ -15,11 +15,11 @@ export class AgileService {
   constructor(private http: HttpClient) {
     this.host = window.location.host;
     if (this.host == "localhost:4200") {
-      this.apiurl = "http://localhost:3000/";
+      this.apiurl = "https://localhost:3000/";
     } else if (this.host == "staging.openebs.io") {
-      this.apiurl = "https://staging.mayadata.io/api/";
+      this.apiurl = "https://workloads.openebs.ci/";
     } else {
-      this.apiurl = "https://openebs.io/api/";
+      this.apiurl = "https://workloads.openebs.ci/";
```

Comment:
@inyee786 Can you verify this agile service url

Reply:
this is correct url "https://openebs.io/api/

Reply:
done